pkg/reconciler/openshift/tektonhub/extension.go

```diff
@@ -404,6 +404,14 @@ func (oe openshiftExtension) SetAuthBaseURL(ctx context.Context, th *v1alpha1.Te
 			return err
 		}
 		th.Status.SetAuthRoute(fmt.Sprintf("https://%s", authRoute))
+
+		secret.StringData = make(map[string]string)
```

Comment:
if one of auth is configured for eg. GitHub this code will always be executed right?
 even if the data is already updated in the secret? 
can we check if url is not set then update or else move on?

Reply:
so you mean check the value of the url, if it is empty then only update the secret ?

Reply:
yes if url is already set in the secret we can skip the update call

Reply:
yeah sounds good 👍🏻

Reply:
will update the  patch

Reply:
updated the patch!